modules/tinymce/src/plugins/media/test/ts/browser/MediaPluginSanityTest.ts

```diff
@@ -1,68 +1,65 @@
-        apis.sSetContent(''),
-        Utils.sOpenDialog(ui),
-        Utils.sSetHeightAndWidth(ui, '300', '300'),
-        Utils.sAssertHeightAndWidth(ui, '300', '300'),
-        Utils.sChangeHeightValue(ui, ''),
-        Utils.sAssertHeightAndWidth(ui, '', '300'),
-        Utils.sPasteSourceValue(ui, 'https://youtu.be/G60llMJepZI'),
-        Utils.sAssertHeightAndWidth(ui, '314', '300'),
-        Utils.sCloseDialog(ui)
-      ]),
-      Log.stepsAsStep('TINY-4857', 'Media: Test embed with XSS attack sanitized', [
-        Utils.sOpenDialog(ui),
-        Utils.sPasteTextareaValue(ui, '<video controls="controls" width="300" height="150"><source src="a" onerror="alert(1)" /></video>'),
-        ui.sClickOnUi('click save button', Utils.selectors.saveButton),
-        Utils.sAssertEditorContent(apis, editor, '<p><video controls="controls" width="300" height="150"><source src="a" /></video></p>')
-      ])
-    ], onSuccess, onFailure);
-  }, {
+describe('browser.tinymce.plugins.media.MediaPluginSanityTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
     plugins: [ 'media' ],
     toolbar: 'media',
-    theme: 'silver',
     base_url: '/project/tinymce/js/tinymce'
-  }, success, failure);
+  }, [ Plugin, Theme ]);
+
+  it('TBA: Embed content, open dialog, set size and assert constrained and unconstrained size recalculation', async () => {
+    const editor = hook.editor();
+    editor.setContent('');
+    await Utils.pTestEmbedContentFromUrl(editor,
+      'www.youtube.com/watch?v=b3XFjWInBog',
+      '<iframe src="https://www.youtube.com/embed/b3XFjWInBog" width="560" height="314" allowFullscreen="1"></iframe>'
+    );
+    await Utils.pTestEmbedContentFromUrl(editor,
+      'http://www.youtube.com/watch?v=b3XFjWInBog',
+      '<iframe src="http://www.youtube.com/embed/b3XFjWInBog" width="560" height="314" allowFullscreen="1"></iframe>'
+    );
+    await Utils.pTestEmbedContentFromUrl(editor,
+      'https://www.youtube.com/watch?v=b3XFjWInBog',
+      '<iframe src="https://www.youtube.com/embed/b3XFjWInBog" width="560" height="314" allowFullscreen="1"></iframe>'
+    );
+    await Utils.pTestEmbedContentFromUrl(editor,
+      'https://www.youtube.com/watch?v=cOTbVN2qZBY&t=30s&index=2&list=PLfQW7NTMsSA1dTqk1dMEanFLovB4-C0FT',
+      '<iframe src="https://www.youtube.com/embed/cOTbVN2qZBY?t=30s&amp;index=2&amp;list=PLfQW7NTMsSA1dTqk1dMEanFLovB4-C0FT" width="560" height="314" allowFullscreen="1"></iframe>'
+    );
+    await Utils.pTestEmbedContentFromUrl(editor,
+      'https://www.google.com',
+      '<video width="300" height="150" controls="controls">\n<source src="https://www.google.com" />\n</video>'
+    );
+    await Utils.pAssertSizeRecalcConstrained(editor);
+    await Utils.pAssertSizeRecalcUnconstrained(editor);
+    await Utils.pAssertSizeRecalcConstrainedReopen(editor);
+  });
+
+  it(`TBA: Test changing source, width and height doesn't delete other values`, async () => {
```

Comment:
Does this need to be in backticks?

```suggestion
  it('TBA: Test changing source, width and height doesn't delete other values', async () => {
```

Reply:
Yes, see `doesn't`.